Source/SwiftLintBuiltInRules/Rules/Lint/FunctionArgumentsSpacingRule.swift

```diff
@@ -0,0 +1,93 @@
+import SwiftSyntax
+
+@SwiftSyntaxRule
+struct FunctionArgumentsSpacingRule: Rule {
+    var configuration = SeverityConfiguration<Self>(.warning)
+
+    static let description = RuleDescription(
+        identifier: "functions_arguments_spacing",
+        name: "Function Arguments Spacing",
+        description: "Remove the space before the first function argument and after the last argument",
```

Comment:
I wonder if this rule should do more than checking the first and the last argument. Can it check all arguments in a function call, so that there is no whitespace before and first and after the last argument and exactly one space after every comma?